### ISSUE TITLE
Needs broader targets

### ISSUE THREAD

**Issue by smudge202** [serilog/serilog-extensions-logging#11]
Currently this package only supports `DNX451` and `DNXCORE50`.

`Microsoft.Framework.Logging` supports a much broader range of TFM's: https://www.nuget.org/packages/Microsoft.Framework.Logging

**Comment by smudge202** [comment]
The package was recently brought inline with `beta6` - just in time for `beta7` more or less! :) However, the project continues to only target dnx profiles; I've only briefly scanned the code but see no reason this can't be expanded to become a superset of the core `Serilog` TFMs. //CC @sblackler

**Comment by nblumhardt** [comment]
:+1:, absolutely.

**Comment by sblackler** [comment]
I've got it building under just "dotnet". Tried to pull that into various projects and it didn't work, so experimenting with expanding the TFM's.

WIP branch: https://github.com/sblackler/serilog-framework-logging/commits/DNX-Updates

**Comment by sblackler** [comment]
I've added explicit TFM's for DNXCORE50 + DNX45.

Having trouble getting it to build inside of Visual Studio, but does build for me okay from the console. I think it's to do with the tooling inside of Visual Studio, specifically the Design Time Host using an older version of the Logging framework. I can't remember how I found this out before, will tweet the chaps tomorrow or drop in chat and ask them.

Proof of concept package:  [Serilog.Framework.Logging.1.0.0-beta7.nupkg](https://onedrive.live.com/redir?resid=B1F50965937053A8!97937&authkey=!AD2x8SQBXq43TKg&ithint=file%2cnupkg). Try and pull it into a project. I've got regular net45 apps running, not tried dnx based ones yet, but should be fine.

Next up is the tests, i'll sort that tomorrow.
